### PR TITLE
Feature/home and quiz styling

### PR DESCRIPTION
**Pull request by nzambonivergara**
### Features
- Created the home page. This will eventually be divided into components. Any input on how to split up the components is welcomed. :)
- Styled both the home page and the quiz page. Resources is not yet created.

### Refactors 
- I refactored how the next and previous buttons were being enabled and disabled in the Quiz component. 

### Issues and Technical Debt
- I need to add error handling to both the home and quiz pages.
- I need to make sure I include aria labels where needed, especially in those places where I have icons.

### Other Comments, Concerns, or Questions
- I also need to break down the Quiz component into smaller components.

### Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 import NavBar from '../NavBar/NavBar';
 import Quiz from '../Quiz/Quiz';
 import './App.css';
 
 const App = () => {
   const [ questions, setQuestions ] = useState([])
   const [ error, setError] = useState('')
-  const [ score, setScore ] = useState(0);
+  const [ score, setScore ] = useState(0)
+  const [ step, setStep ] = useState(0)
```
Comment (nzambonivergara) [inline src/components/App/App.js:12]: I should rename step to factNumber to be more descriptive.

File: src/components/App/App.js [hunk 2]
```diff
@@ -42,10 +45,50 @@ const App = () => {
             render={() => {
               return (
                 <>
-                  <h2>Take the Sexual Health Myth Debunking Quiz!</h2>
-                  <Link to="/quiz">YES!</Link>
+                  <div className="quiz-call-to-action">
```
Comment (nzambonivergara) [inline src/components/App/App.js:48]: Any input on how to break this down into components is welcomed or should I have one big component called Home?

File: src/components/App/App.js [hunk 3]
```diff
@@ -42,10 +45,50 @@ const App = () => {
...
+                        className="arrow-button"
+                        onClick={() => setStep(step + 1)}
+                      >
+                        <span class="material-icons">
+                          arrow_forward
+                        </span>
+                      </button>
+                    </div>
+                  </div>
+                  {
```
Comment (nzambonivergara) [inline src/components/App/App.js:83]: I will remove this form for now, I am not sure how to make this functional with my current API.

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -12,16 +12,19 @@ const NavBar = () => {
 
   return (
     <nav className="nav-bar">
-      <div class="container" id="menu" onClick={toggleMenu}>
-        <div class="bar1"></div>
-        <div class="bar2"></div>
-        <div class="bar3"></div>
+      <div className="container" id="menu" onClick={toggleMenu}>
```
Comment (nzambonivergara) [inline src/components/NavBar/NavBar.js:15]: I think I need an aria label here for the role of the hamburger menu.

File: src/components/App/App.css [hunk 5]
```diff
@@ -1,21 +1,113 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.retake-button,
```
Comment (EricSergeant) [inline src/components/App/App.css:68]: You've got cleanly-written CSS here, and way these are grouped make sense to me (even as someone not familiar with the code).  As a stretch goal, you could always use SCSS as well to further bundle the code.

File: src/components/App/App.js (hunk 1 above)
Comment (EricSergeant) [reply src/components/App/App.js:12]: I don't see "step" in the code here, but if it's a reference to each of your question facts, then yes I'd say "factNumber" is more descriptive.

File: src/components/NavBar/NavBar.js (hunk 4 above)
Comment (EricSergeant) [reply src/components/NavBar/NavBar.js:15]: I haven't tried a hamburger menu yet ( I might for this project).  You could always run a screenreader through the app and see where it's not describing as you want.

File: src/components/Quiz/Quiz.css [hunk 6]
```diff
@@ -1,3 +1,85 @@
+.quiz-form {
```
Comment (EricSergeant) [inline src/components/Quiz/Quiz.css:1]: Good use of responsive-friendly measurements here (vh, rem, flex).  This should make it much easier to resize for different viewers/screen sizes.

File: src/components/Quiz/Quiz.js [hunk 7]
```diff
@@ -4,111 +4,132 @@ import { Link } from 'react-router-dom';
...
       )
-    } else if (userAnswers[step] !== undefined && userAnswers[step] !== questions[step].correctAnswer) {
+    } else if (userAnswers[questionNumber] !== undefined && userAnswers[questionNumber] !== questions[questionNumber].correctAnswer) {
       return (
-        <>
-          <p>❌ Let’s look at that one again!</p>
-          <p>{questions[step].fact}</p>
-        </>
+        <div className="fact-container">
+          <p className="result">❌ Let’s look at that one again!</p>
```
Comment (EricSergeant) [inline src/components/Quiz/Quiz.js:48]: I like the "let's look at that one again" language.  Much better than saying "wrong!" and it redirects the focus back to the point that this is about learning for the app user.

File: src/components/Quiz/Quiz.js [hunk 8]
```diff
@@ -4,111 +4,132 @@ import { Link } from 'react-router-dom';
...
+              </button>
+            </div>
+            <p>You have completed {userAnswers.length / questions.length * 100}% of the quiz!</p>
+          </form>
           { showResult() }
-          <p>You have completed {userAnswers.length / questions.length * 100}% of the quiz!</p>
-        </form>
+        </>
       )
       :
```
Comment (EricSergeant) [inline src/components/Quiz/Quiz.js:126]: This was the longest ternary I've seen!  I honestly never think to use them in this way, and I think it's great.

**Review by EricSergeant — APPROVED** [review]
This all looks very good, Natalia!  Your styling is really clean and your use of hooks is shows a very clear method to making the code as developer-friendly as possible.  I'm looking forward to trying out the deployed app!

File: src/components/NavBar/NavBar.js (hunk 4 above)
Comment (nzambonivergara) [reply src/components/NavBar/NavBar.js:15]: Great idea, will do, thank you!

**Pull request merged**